siocli/lib/SqlDump.cpp

```diff
@@ -362,7 +362,7 @@ std::vector<ColumnConstaint> receiveColumnConstraintsList(io::IoBase& connection
 
         if (rowLength == 0) break;
 
-        ColumnConstaint constraint;
+        ColumnConstraint constraint = ColumnConstraint();
```

Comment:
Please leave this line of code as before. Instead, please introduce default constructor in the each new `struct` you've intriduced and initialize member variables of fundamental types and enums in them.